src/test/java/lotto/domain/lotto/LottoTicketTest.java

```diff
@@ -0,0 +1,59 @@
+package lotto.domain.lotto;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class LottoTicketTest {
+
+  private List<LottoNumber> winning;
+
+  @BeforeEach
+  void setUp() {
+    winning = List.of(
+        LottoNumber.of(1), LottoNumber.of(2),
+        LottoNumber.of(3), LottoNumber.of(4),
+        LottoNumber.of(5), LottoNumber.of(6));
+  }
+
+  @Test
+  void lottoTicketNotHas6LottoNumbersThrowsException() {
+    assertThatThrownBy(() ->
+        new LottoTicket(List.of(1))).isInstanceOf(IllegalArgumentException.class);
+
+    assertThatThrownBy(() ->
+        new LottoTicket(List.of(1, 2, 3, 4, 5))
```

Comment:
```suggestion
        new LottoTicket(1, 2, 3, 4, 5)
```
가변 인자를 받는 생성자를 추가한다면 위와 같이 테스트 코드를 작성할 수 있지 않을까?